src/main/java/duke/Message.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * Message to be displayed on screen with a user and exit status.
+ */
+public class Message {
+    public final String message;
+    public final boolean shouldExit;
+    public final User user;
+
+    /**
+     * Represents the list of valid users.
+     */
+    public enum User {
+        USER,
+        DUKE
+    }
+
+    /**
+     * Creates a message with the given text, exit status, and user.
+     * @param message message to be displayed
```

Comment:
Javadoc comments require empty line between description and parameter section, i noticed the same issue in several other places too